schema.graphql

```diff
@@ -815,6 +873,21 @@ input DeleteProjectInput {
   projectId: ID!
 }
 
+input WidgetLocationInput {
+  zone: String!
+  section: String!
+  area: String!
+}
+
+input WidgetLayoutInput {
+  extended: Boolean
+	Location: WidgetLocationInput!
+	newLocation: WidgetLocationInput
+	oldIndex: Int
+	newIndex: Int
+  align: String
+}
+
 input AddWidgetInput {
   sceneId: ID!
```

Comment:
Since widgets have a default location I thought that wouldn't be needed as a widget would be always added to the default location first. 
Do you think we will have UI in the future where the user somehow specifies where the widget will go when they enable the widget?

Reply:
If I were you, I would add a field that allows you to specify the location from the beginning for that possibility. But it is not mandatory.